### PR TITLE
Move addStream test to webrtc/legacy

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

**Comment by sideshowbarker** [comment]
Stability checking is timing out for Chrome in Travis https://travis-ci.org/web-platform-tests/wpt/jobs/454739870

**Comment by youennf** [comment]
> Stability checking is timing out for Chrome in Travis https://travis-ci.org/web-platform-tests/wpt/jobs/454739870

Checking the modified test in Chrome, it seems to timeout before and after my change.
But I wouldn't expect that to cause the stability check to time out.
Let's give it another try.

**Comment by sideshowbarker** [comment]
> > Stability checking is timing out for Chrome in Travis https://travis-ci.org/web-platform-tests/wpt/jobs/454739870
> 
> Checking the modified test in Chrome, it seems to timeout before and after my change.
> But I wouldn't expect that to cause the stability check to time out.
> Let's give it another try.

This should be merged anyway after it gets review-approved, even if the stability checker is failing — because the timeout problem isn’t specific to this PR. See https://github.com/web-platform-tests/wpt/issues/7660 and https://github.com/web-platform-tests/wpt/pull/14033.

**Review by soareschen — APPROVED** [review]

**Pull request merged**
